stable/netsil/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: netsil
+version: 1.5.3
```

Comment:
`appVersion` which should be what `version` currently is.

Reply:
done